README.md

```diff
@@ -106,19 +91,15 @@ const MyPlaidComponent = () => {
 
 `PlaidLink` wraps the view you provide as a child with a `Pressable` component and intercepts the `onPress` event.
 
-### OAuth requirements
+#### OAuth requirements
 
-For Link Token based OAuth support, you must configure your `link_token` with a `redirect_uri` to support OAuth on iOS. Other than setting the `redirect_uri`, which must be a universal link, when you create the `link_token` no further configuration is required. Notably, no props are required on the React Native side.
+For Link Token based OAuth support, you must configure your `link_token` with a [redirect_uri](https://plaid.com/docs/api/tokens/#link-token-create-request-redirect-uri) to support OAuth on iOS. On Android you need to register your package name as described above. Other than setting the `redirect_uri`, which must be a universal link, when you create the `link_token` no further configuration is required. Notably, no props are required on the React Native side.
```

Comment:
I think the OAuth requirements in the Android section should be pulled down to here so that they are side by side and it's clear that a link token request for a RN app should have either a `redirect_uri` (iOS) or a `package_name` (Android) specified.

Reply:
I moved all the OAuth down to this section and called out your comment like this 

>##### Link Token OAuth Requirements

- On iOS you must configure your `link_token` with a [redirect_uri](https://plaid.com/docs/api/tokens/#link-token-create-request-redirect-uri) to support OAuth. When creating a `link_token` for initializing Link on Android, `android_package_name` must be specified and `redirect_uri` should be left blank.

- On Android you must configure your `link_token` with an [android_package_name](https://plaid.com/docs/api/tokens/#link-token-create-request-android-package-name) to support OAuth. When creating a `link_token` for initializing Link on iOS, `android_package_name` must be left blank and `redirect_uri` should be used instead.